### ISSUE TITLE
Network requests by axios is blocked by CORS

### ISSUE THREAD

**Issue by yooneskh** [neutralinojs/neutralinojs#479]
**Expected Behavior**  
Network calls (axios or ...) should not be blocked by CORS as this is a desktop application.
  
**Actual Behavior**  
Netowrk calls by axios is blocked by CORS.

**Steps to Reproduce the Problem**  
 1. npm i axios
 2. axios.get('https://google.com')
 3. CORS error in network

**Specifications**  
* NeutralinoJs Version: 2.3.0
* NeutralinoJs CLI version: 4.1.1
* Platform: Fedora Linux 43

**Comment by Mrgove10** [comment]
+1 on this...

**Comment by F3WS** [comment]
How can we get around this?

**Comment by shalithasuranga** [comment]
Thanks for reporting and discussing this. I am closing this one as a duplicate of the main case: https://github.com/neutralinojs/neutralinojs/issues/586
